R/object_name_linter.R

```diff
@@ -45,7 +45,9 @@ object_name_xpath <- local({
 #'
 #' @param styles A subset of
 #'   \Sexpr[stage=render, results=rd]{lintr:::regexes_rd}. A name should
-#'   match at least one of these styles.
+#'   match at least one of these styles. The `"symbols"` style refers to
+#'   names containing only non-alphanumeric characters, e.g., `%+%` from
```

Comment:
maybe `*only*` to emphasize that e.g. `%in%` is not covered?

Reply:
Sure -- I'm not totally sure what the appropriate grammar is, but how does this sound? https://github.com/r-lib/lintr/pull/2001/commits/521f248e92db4650908dc8a952bf6009d90e56b0